src/pynwb/data/nwb.namespace.yaml

```diff
@@ -46,4 +46,4 @@ namespaces:
   - source: nwb.retinotopy.yaml
     title: Retinotopy
     doc: This source module contains neurodata_type for retinotopy data.
-  version: 2.0.1
+  version: 2.0.2
```

Comment:
@oruebel Is that the version written to the NWB file?

Using latest master I get

```sh
$ h5dump icephys_example.nwb  | grep -A30 version | head -n11
   ATTRIBUTE "nwb_version" {
      DATATYPE  H5T_STRING {
         STRSIZE H5T_VARIABLE;
         STRPAD H5T_STR_NULLTERM;
         CSET H5T_CSET_UTF8;
         CTYPE H5T_C_S1;
      }
      DATASPACE  SCALAR
      DATA {
      (0): "2.0b"
      }

thomas@thomas-win7-x64 MINGW64 /e/projekte/pynwb (master)
$ git nn
8f472d6c (/#936 fast epoch to df (#937), 2019-05-16)
```

Reply:
Good catch. Looks like the ``nwb_version`` attribute is currently hard-coded in the specification of nwb-file. I've updated the schema accordingly for now, but it should really be populated from the version indicated in the namespace yaml. Can you file a corresponding issue for this. A separate (but related) issue is that we should also store the version of the namespace with the namespace attribute for the types.

Reply:
Will do.